client/src/components/BallotActivityList.vue

```diff
@@ -1,5 +1,8 @@
 <script lang="ts" setup>
 import BallotActivity from "./BallotActivity.vue";
+import { ref } from 'vue';
+
+const isRtl = ref(document.getElementsByTagName("html")[0].dir === "rtl");
```

Comment:
What happens if I change language or direction while the page is in memory? I don't believe that this will update it self correctly.

Reply:
It won't anyway since the tooltip plugin we're using does not support a `position` change without a page reload. So that's why I didn't use a computed value here.

In case there's no space for the tooltip to be displayed in that position, it will show on top/the other side, which is what will happen if you load the page with a language and then change it. This modification will at least work when you're already coming from an already localised URL.

I could force a page reload on language switch but I don't think that's a good user experience... and definitely not needed for just a tooltip, especially if we know it will work and be accessible/readable anyway.

Reply:
I'm not sure I follow your logic here. You want to refrain from making this responsive to direction changes because the tooltips (which are not a part of this PR) are not responsive to direction changes. Is this correctly understood?

Reply:
The tooltips [TECH-2396] are the main focus of this PR. That line is only used for setting the initial position where a tooltip will open and, because that param won't update the tooltip behaviour, it is not necessary to have it reactive. 

I can make it to be computed so the variable will change every time the language is changed, but it won't have any effect on the UI. Or I can just remove that addition from the PR and make the tooltips open always to the right no matter what (and, if there's no space for it like in Arabic, it will just move to wherever it has enough space to show).